### PR TITLE
Add support for json extension

### PR DESCRIPTION
**Pull request by fatbeard2**
Hi, thanks for the library.

This PR adds ability to inject digests in json files. It allows user to generate an assets map and use it as a centralised container of all digests(this is my usecase). I know that it's not exactly aligned with the library name, but I thought it could be a good addition.

### CODE REVIEW

File: example/pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: built_html_example
 version: 1.0.0
-description: Example to build *.html files from *.template.html source.
+description: Example to build *.html, *json files from *.template.html and *.template.html source.
```
Comment (jakemac53) [inline example/pubspec.yaml]:
```suggestion
description: Example to build *.html, *json files from *.template.html and *.template.json source.
```

File: pubspec.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: built_html
 version: 0.1.3
```
Comment (jakemac53) [inline pubspec.yaml]:
```suggestion
version: 0.1.4
```

And then add a changelog entry please :)

**Review by jakemac53 — COMMENTED** [review]
Ya its a little bit weird given the name but its easy enough so I am fine with adding this just some minor suggestions, thanks!

**Comment by fatbeard2** [comment]
@jakemac53 Thanks for the quick response. I commited suggested changes.

**Comment by jakemac53** [comment]
published the new version, thanks!

**Pull request merged**